app/assets/json/translations/locale-en.json

```diff
@@ -221,7 +215,6 @@
         "INSTRUCTIONS_P2": "Just upload valid proof of this address and <strong>we'll also give you Live in San Francisco Preference.</strong>",
         "PREFERENCE": {
             "DESCRIPTION": "<strong>Neighborhood Resident Housing Preference</strong> means at least one household member lives within the same Supervisorial District or within a 1/2 mile buffer of the project.",
-            "NAME": "Neighborhood Resident Housing Preference",
```

Comment:
The names of the preferences are gone too.. E.g. these are found in `preferencesSummary`.

It looks like translations is deleting the strings that don't exist anymore in the html files (from running `grunt translations`) , but do so incorrectly coz they are getting translated in the angular code 🤔 I think what we did before for this was to undo the unnecessary deletions.